rules/integrations/aws/credential_access_rapid_secret_retrieval_attempts_from_secretsmanager.toml

```diff
@@ -116,5 +148,9 @@ reference = "https://attack.mitre.org/tactics/TA0006/"
 
 [rule.threshold]
 field = ["user.id"]
+value = 1
+[[rule.threshold.cardinality]]
+field = "aws.cloudtrail.request_parameters"
```

Comment:
Based on [docs](https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_GetSecretValue.html#API_GetSecretValue_RequestSyntax) there could be `VersionId` or `VersionStage` in the request. Maybe we use `aws.cloudtrail.flattened.request_parameters.secretId` instead?